addon/components/imgix-photo.js

```diff
@@ -72,7 +64,40 @@ export default Component.extend({
     return `${url}?${queryParams}`;
   }),
 
+  isConversationNeeded: match('url', /\.(heic)$/),
+
   width: and('includeDimensions', 'params.w'),
 
   height: and('includeDimensions', 'params.h'),
+
+  defaultParams: computed('isConversationNeeded', function() {
+    let { isConversationNeeded } = this;
+
+    let defaultParams = { dpr: window.devicePixelRatio };
+
+    if (isConversationNeeded) {
+      defaultParams.fm = 'jpg';
+    }
+
+    return defaultParams;
+  }),
+
+  finalParams: computed('params.[]', 'defaultParams.[]', function() {
+    let {
+      defaultParams,
+      params,
+    } = this;
+
+    if (params) {
+      return { ...defaultParams, ...params };
+    } else {
+      return { ... defaultParams };
```

Comment:
Is this space correct?

Regardless, if its a mistake or not, We should probably add a simple Integration test for when _no_ params are passed in